documentation-website/Writerside/topics/DSL-Querying-data.topic

```diff
@@ -7,16 +7,24 @@
     <show-structure for="chapter,procedure" depth="2"/>
     <chapter id="where-expression">
         <title>Working with <code>where</code> expressions</title>
-        <p>The query expression <code>where</code> expects a boolean operator (<code>Op&lt;Boolean&gt;</code>).
-            We've split the allowed conditions in different categories:</p>
+        <p>
+            The <code>where</code> expression is a core component of building queries in Exposed, allowing you to
+            filter data by specifying conditions. A <code>where</code> expression expects a boolean operator
+            (<code>Op&lt;Boolean&gt;</code>), which evaluates to either <code>true</code> or <code>false</code>.
+        </p>
+        <p>We’ve categorized the available conditions into the following groups:</p>
         <list>
             <li><a anchor="basic">Basic conditions</a>.</li>
             <li><a anchor="logical-conditions">Logical conditions</a>.</li>
-            <li><a anchor="check-for-a-match-in-a-pattern">Conditions that check for a match in a pattern</a>.</li>
-            <li><a anchor="check-for-a-match-in-a-range">Conditions that check for a match in a range</a>.</li>
-            <li><a anchor="check-for-a-match-in-a-collection">Conditions that check for a match in a collection</a>.</li>
+            <li><a anchor="pattern-matching-conditions">Pattern-matching conditions</a>.</li>
+            <li><a anchor="range-conditions">Range conditions</a>.</li>
+            <li><a anchor="collection-conditions">Collection conditions</a>.</li>
```

Comment:
**Minor:** Maybe it's just me, but it looks odd to see a list-item hyperlink with a `.` at the end, when it's not part of a sentence. I'd consider removing the periods entirely, unless there's some rule about keeping them.

Reply:
You are right, the periods do look weird. Technically they shouldn't be there as the texts are not complete sentences. Removed ✅